srcpkgs/pymol/template

```diff
@@ -0,0 +1,24 @@
+# Template for 'pymol'
+pkgname=pymol
+version=2.1.0
+revision=1
+wrksrc=${pkgname}
+build_style="python3-module"
+hostmakedepends="msgpack-devel python3-numpy freetype-devel libfreeglut-devel glew-devel python3-devel libxml2-devel python3-PyQt5"
+depends="freetype glew libfreeglut python3-numpy tcsh python3-pmw python3-tkinter python3-PyQt5"
+short_desc="PyMOL molecular visualization system"
+homepage="https://pymol.org/"
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+license="pymol"
+only_for_archs="i686 x86_64"
+distfiles="${SOURCEFORGE_SITE}/project/pymol/pymol/${version/.*/}/pymol-v${version}.tar.bz2
+https://c.fsdn.com/allura/p/pymol/icon>pymol.png"
+skip_extraction="pymol.png"
+checksum="7ae8ebb899533d691a67c1ec731b00518dea456ab3e258aa052a65c24b63eae2
+0ea81faaf336becc669a193777d0dca55475d303d1236b57df25cf67ff7c2bcd"
+
+post_install() {
+        vlicense LICENSE
```

Comment:
indent with tabs

Reply:
I have. In VSCode I've removed the indentation on that line, and indented with a tab instead. That was the first thing I did in that line.

Reply:
OK I opened it up in leafpad this time and redid with tabs, hopefully this time we'll see this fixed.